### PR TITLE
Improve message validation check

### LINKED ISSUE #3: Improve message validation check
When a message is entered into the controller,
The controller calls the following logic to verify the message.

`ReqMsgVerifier.validateRegOptionRequest(regOptionRequest);`

We expect this logic to be improved with `Spring Boot Bean Validation`.

### PR DESCRIPTION
**Pull request by seachicken**
# What is this PR for?

## Overview or reasons
Resolves #3

## Tasks
- Replace to Bean Validation
- Add error handling

### CODE REVIEW

File: server/src/main/java/com/linecorp/line/auth/fido/fido2/server/util/ReqMsgVerifier.java [hunk 1]
```diff
@@ -1,181 +0,0 @@
...
-        if (regOptionRequest.getRp() == null ||
-                regOptionRequest.getUser() == null) {
-            throw new FIDO2ServerRuntimeException(InternalErrorCode.INVALID_REQUEST_BODY);
-        }
-
-        if (regOptionRequest.getUser().getId() == null) {
-            throw new FIDO2ServerRuntimeException(InternalErrorCode.INVALID_REQUEST_BODY, "userId should not be null");
-        }
-        if (!(regOptionRequest.getUser().getId().length() > 0 && regOptionRequest.getUser().getId().length() < 64)) {
-            throw new FIDO2ServerRuntimeException(InternalErrorCode.INVALID_REQUEST_BODY, "userId length must be between 1 and 64 bytes");
```
Comment (seachicken) [inline server/src/main/java/com/linecorp/line/auth/fido/fido2/server/util/ReqMsgVerifier.java:172]: Is it correct that just 64 bytes are an error?

Comment (kj84park) [reply server/src/main/java/com/linecorp/line/auth/fido/fido2/server/util/ReqMsgVerifier.java:172]: Yes, that's right.
You can find the rules here.
https://www.w3.org/TR/webauthn/#sctn-createCredential
`If the length of options.user.id is not between 1 and 64 bytes (inclusive) then return a TypeError.`

BTW, I'm planning to proceed with the review soon. 
Thanks!

Comment (seachicken) [reply server/src/main/java/com/linecorp/line/auth/fido/fido2/server/util/ReqMsgVerifier.java:172]: Thank you for reviewing 😃

>If the length of options.user.id is not between 1 and 64 bytes (inclusive) then return a TypeError.

It looks like 64 bytes are not inclusive.
```java
regOptionRequest.getUser().getId().length() < 64
```
I may be wrong, but I think the following is correct.
```java
regOptionRequest.getUser().getId().length() <= 64
```

Comment (kj84park) [reply server/src/main/java/com/linecorp/line/auth/fido/fido2/server/util/ReqMsgVerifier.java:172]: Oh yes, I think your opinion is correct.

Comment (seachicken) [reply server/src/main/java/com/linecorp/line/auth/fido/fido2/server/util/ReqMsgVerifier.java:172]: https://github.com/line/line-fido2-server/pull/20/files#diff-f579719df9002ef23bdab8a28a4d4dc7986c3ceec90b9f809ad5fef98af6e94dR30

Ok, I will change the maximum length

`@Length(min = 1, max = 63)` -> `@Length(min = 1, max = 64)`

**Review by kj84park — APPROVED** [review]
Everything looks good!
Thank you for your contribution!

**Comment by kj84park** [comment]
> Everything looks good! Thank you for your contribution!



> Everything looks good! Thank you for your contribution!

**Comment by seachicken** [comment]
The signing of CLA is done!

**Pull request merged**